site/app/templates/course/CourseMaterials.twig

```diff
@@ -297,8 +298,80 @@
     function setNewDateTime(me, path) {
         // pass filename to server to record the new date and time of the file to be released
         var newDateTime = $(me).val();
-
         changeNewDateTime(path, newDateTime);
+        var today = new Date();
+        //to properly compare dates/times they need to match in format
+        //not getting it from the picker leaves out the zero before single digits
+        if(today.getMonth()+1<10){
```

Comment:
Are you using these formated strings anywhere, if so lines lines 305 to 340 can be replaced with :
```js
var now = new Date();

function pad(str){
  return ('0' + str).slice(-2);
}


var date = now.getFullYear() + "-"+ pad(now.getMonth()) + "-" + pad(now.getDay());
var time = pad(now.getHours()) + ":" + pad(now.getMinutes()) + ":" + pad(now.getSeconds())

var currentDT = date+' '+time;
var neverDT = (now.getFullYear()+10)+'-'+pad(now.getMonth())+'-'+pad(now.getDay())+' '+time;
```

Reply:
This may be of help too. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart

Reply:
ah I'll get right on that, this sounds dumb, but I literally didn't know js had  a pad option... ;A;